rig/machine_control/boot.py

```diff
@@ -0,0 +1,160 @@
+"""Boot constructs for a SpiNNaker machine.
+
+Implementation is reconstructed from a Perl implementation which forms a
+significant part of the documentation for this process.
+"""
+from . import struct_file
+import socket
+import struct
+import time
+
+# Specifies the size of packets that should be sent to SpiNNaker to boot the
+# board, and to which port.
+BOOT_MAX_BLOCKS = 32  # 32 kiB limit in DTCM
+BOOT_WORD_SIZE = 256
+BOOT_BYTE_SIZE = 1024
```

Comment:
These guys aren't quite clear. Isn't one a function of the other, anyway so:

    BOOT_BYTE_SIZE = 1024  # Size of the boot image in bytes
    BOOT_WORD_SIZE = BOOT_BYTE_SIZE / 4